components/Feature/Feature.js

```diff
@@ -3,112 +3,148 @@ import Link from 'next/link'
-  constructor(){
-    super();
-    this.renderSuccessCase = this.renderSuccessCase.bind(this);
-  }
-
-  renderSuccessCase(successCase){
-    if(successCase){
-      return(
-        <Link href={`${successCase.href}`}>
-          <a className="link--primary">{successCase.text}</a>
-        </Link>
-      )
-    }
-    return null
-  }
-
-  render() { 
-    const { features } = this.props;
-    return(
-      features.map((feature, index) => {
-        return(
-          <section key={index} className={`feature-section ${feature.hasColorBackground ? "color-background" : ""}`}>
-            <div className="feature">
-              <Title
-                title={feature.title}
-                subtitle={feature.description}
-              />
-              {this.renderSuccessCase(feature.successCase)}
-              <div className="feature-item--container">
-                {feature.featureItems.map((item, index) => <FeatureItem
-                  key={index}
-                  icons={item.icon}
-                  title={item.title}
-                  description={item.description}
-                  button={item.button}
-                />)}
-              </div>
-            </div>
-            <style jsx>
-            {`
-              {/* STYLES FOR MOBILE */}
-              @media only screen and (max-width: 750px) {
-                .feature-section {
-                  padding: 30px;
-                }
-                .feature-section.color-background {
-                  background: #f0f9fe;
-                }
-                :global(a.link--primary) {
-                  color: rgb(6, 125, 247);
-                  text-decoration: none;
-                  margin-bottom: 24px;
-                  display: inline-block;
-                }
-                .feature :global(a.link--primary) {
-                  margin-bottom: 46px;
-                }
-              }
-    
-              {/* STYLES FOR DESKTOP */}
-              @media only screen and (min-width: 751px) {
-                .feature-section {
-                  border-bottom: 1px solid #dadedf;
-                }
-                .feature-section.color-background {
-                  background: #f0f9fe;
+const Feature = ({ features }) => {
+  return (
+    <>
+      {
+        features.map((feature, index) => {
+          const { title, description, hasColorBackground, successCase, featureItems, image, layout } = feature;
+          return (
+            <section key={index} className={`feature-section ${hasColorBackground ? "color-background" : ""}`}>
+              <div className="feature">
+                <Title
+                  title={title}
+                  subtitle={description}
+                />
+                {successCase ?
+                  <Link className="link--display" href={`${successCase.href}`}>
+                    <a className="link--primary">{successCase.text}</a>
+                  </Link>
+                  : <></>
```

Comment:
Para este caso se puede usar este ternario:
```suggestion
                {successCase &&
                  <Link className="link--display" href={`${successCase.href}`}>
                    <a className="link--primary">{successCase.text}</a>
                  </Link>
```

Basicamente lo que hace es, si `successCase` es truthy (https://developer.mozilla.org/es/docs/Glossary/Truthy), renderiza el componente. En caso de que sea falsy, no.

Reply:
Buenísimo, gracias por la info!